src/server/create.go

```diff
@@ -2,44 +2,32 @@ package main
 
 import (
 	"encoding/json"
-	"log"
 	"net/http"
 	"person"
 )
 
 func Create(w http.ResponseWriter, r *http.Request) {
-	// Define the request we are expecting
-	req := struct {
-		Obj person.Model `json:"person"`
-	}{}
-	// Parse data from request
+
+	var req Request
 	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
-		log.Print(err)
-		http.Error(w, err.Error(), http.StatusBadRequest)
+		HTTPError(w, err, http.StatusBadRequest)
 		return
 	}
-	// Attempt to create new object in database
-	err := person.Create(&req.Obj)
-	if err != nil {
-		log.Print(err)
-		http.Error(w, err.Error(), http.StatusBadRequest)
+
+	if err := person.Create(&req.Obj); err != nil {
+		HTTPError(w, err, http.StatusBadRequest)
 		return
 	}
-	// Define what we want our response to look like
-	res := struct {
-		Error  error        `json:"error"`
-		Object person.Model `json:"object"`
-	}{
+
+	output, err := json.Marshal(Response{
```

Comment:
Sometimes it's perfectly fine to inline a struct or other type simply because you're not going to reuse it.